pulp_2to3_migration/app/plugin/docker/migrator.py

```diff
@@ -0,0 +1,270 @@
+import json
+
+from . import pulp2_models
+
+from .pulp_2to3_models import (
+    Pulp2Blob,
+    Pulp2Manifest,
+    Pulp2ManifestList,
+    Pulp2Tag,
+)
+
+from pulp_docker.app.models import (
+    Blob,
+    BlobManifest,
+    Manifest,
+    ManifestListManifest,
+    Tag,
+)
+
+from pulp_2to3_migration.app.constants import NOT_USED
+from pulp_2to3_migration.app.plugin.api import (
+    ContentMigrationFirstStage,
+    DeclarativeContentMigration,
+    Pulp2to3PluginMigrator,
+    RelatePulp2to3Content,
+)
+
+from pulpcore.plugin.stages import (
+    ArtifactSaver,
+    ContentSaver,
+    DeclarativeArtifact,
+    DeclarativeContent,
+    Stage,
+    QueryExistingArtifacts,
+    QueryExistingContents,
+)
+
+
+class DockerMigrator(Pulp2to3PluginMigrator):
+    """
+    An entry point for migration the Pulp 2 Docker plugin to Pulp 3.
+    """
+    pulp2_plugin = 'docker'
+    pulp2_content_models = {
+        'docker_blob': pulp2_models.Blob,
+        'docker_manifest': pulp2_models.Manifest,
+        'docker_manifest_list': pulp2_models.ManifestList,
+        'docker_tag': pulp2_models.Tag,
+    }
+    pulp2_collection = 'units_docker_manifest'
+    # will be renamed to pulp_container
+    pulp3_plugin = 'pulp_docker'
+    content_models = {
```

Comment:
I just realised that we rely on the order of the keys here which is not good because `dict` doesn't guarantee the order.
Let's use OrderedDict here to be sure. https://docs.python.org/3/library/collections.html#collections.OrderedDict

Reply:
good catch, thanks!